gremlin-go/cucumber/go.mod

```diff
@@ -0,0 +1,41 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+module gremlin-godog-tests
+
+go 1.17
+
+require (
+	github.com/cucumber/godog v0.12.4
+	github.com/lyndonb-bq/tinkerpop/gremlin-go v0.0.0-20220131225152-54920637bf94
```

Comment:
Is this pointing to the right place? How is this generated/determined? I feel like if a file with this specific version is committed three's an oversight here.

Reply:
I'm not entirely sure how this is set up, @lyndonbauto I believe this was added by your latest commits?

Reply:
Yeah this is just an easy way to replace with local version.